AppleStore/AppleStore/ProductViewController.swift

```diff
@@ -9,12 +9,28 @@ import UIKit
 
 final class ProductViewController: UIViewController {
     
-    // MARK: - Private Properties
-    let productImageView = UIImageView()
+    // MARK: - Public Properties
+    var productImageView: [UIImage?] = []
```

Comment:
```suggestion
  private  var productImageView: [UIImage?] = []
```
передавай массив имен а в этом vc уже присваивай